frontend/ui/src/components/GeneralIconButton/styles.ts

```diff
@@ -0,0 +1,28 @@
+import { styled } from "@mui/material/styles";
+import { IconButton } from "@mui/material";
+
+export const StyledIconButton = styled(IconButton)`
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 16px;
+  gap: 8px;
+
+  width: 3.5rem;
+  height: 3.5rem;
+
+  background: #451d53;
+  color: white;
+
+  box-shadow: 0px 3px 5px -1px rgba(0, 0, 0, 0.2),
```

Comment:
I don't think you need to redefine all this styles. All you need to do is use IconButton size='large' and set background color from theme :)